### PR TITLE
fix: selenium m1 support and deprecate own vnc

### PR DESCRIPTION
**Pull request by fschade**
## Description
the shipped selenium chrome was not part of supported browsers for wen anymore, this bumps the used selenium image (arm only) and removes the dedicated vnc. 

VNC is now part of the selenium image!
https://github.com/seleniumhq-community/docker-seleniarm

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -92,6 +92,7 @@ services:
     ports:
       - 4444:4444
       - 5900:5900
+      - 7900:7900
```
Comment (dschmidt) [inline docker-compose.yml:95]: Does it work to expose port 7900 on 6080 (additionally) ?

Comment (fschade) [reply docker-compose.yml:95]: lets give it a try 🤣

Comment (fschade) [reply docker-compose.yml:95]: works, done

**Comment by fschade** [comment]
@grimmoc, can you give it a try and confirm it works for ya?
@dschmidt can you give it a try too? not sure if selenium/standalone-chrome:104.0-20220812 ships vnc too, hard to find informations as a m1 user ;)

https://hub.docker.com/layers/selenium/standalone-chrome/104.0-20220812/images/sha256-d0f6a93b90d7cb55f98431598d259ad67335b6114330009aee4d86881f77223b?context=explore

**Comment by dschmidt** [comment]
> @grimmoc, can you give it a try and confirm it works for ya?
> @dschmidt can you give it a try too? not sure if selenium/standalone-chrome:104.0-20220812 ships vnc too, hard to find informations as a m1 user ;)
> 
> https://hub.docker.com/layers/selenium/standalone-chrome/104.0-20220812/images/sha256-d0f6a93b90d7cb55f98431598d259ad67335b6114330009aee4d86881f77223b?context=explore

I can, in two weeks... 😏

**Review by JammingBen — APPROVED** [review]

**Review by lookacat — APPROVED** [review]

**Pull request merged**